charts/proxmox-csi-plugin/templates/node-deployment.yaml

```diff
@@ -125,6 +128,9 @@ spec:
           hostPath:
             path: /sys
             type: Directory
+        {{- with .Values.extraVolumes }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
```

Comment:
Node daemonset does not call proxmox directly.
I think the extra volumes does not need here...

Thanks.

Reply:
Tested it and everything seems to work fine.
I squashed all the commits into a single one with all the changes.

Thanks!